test/src/missle_launcher_test.cpp

```diff
@@ -15,12 +17,35 @@ class MissleLauncherTest : public ::testing::Test {
     }
 };
 
+TEST(MissileLauncherTest, team_set) {
+    MissileLauncher friendly_launcher = MissileLauncher(FRIENDLY);
+    friendly_launcher.team() == FRIENDLY;
```

Comment:
What does this do? Did you mean to write `if (friendly_launcher.team() == FRIENDLY)`?  and make it an assert or EXPECTEQ?